### PR TITLE
Development header homepage

### PR DESCRIPTION
**Pull request by monicah79**
created home-page
created header
added styling

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="main-page">
```
Comment (jasonfelice) [inline index.html:11]: - [ ] For semantic reasons, please remove this `div` tag. Nest `header` & `section` directly inside the `body`. 😄

File: index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        <header class="header">
+            <a href="">My Logo</a>
+            <img src="Images/Menu-icon.svg" alt="">
```
Comment (jasonfelice) [inline index.html:14]: - [ ] It'll be semantically better to wrap the hamburger inside the `nav` tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
```
Comment (jasonfelice) [inline index.html:10]: - [ ] Kindly reset the margin and padding in your CSS to fix the highlighted white space. Check the following screenshot.
```
* {
  margin: 0;
  padding: 0;
}
```
image

**Review by jasonfelice — CHANGES_REQUESTED** [review]
Hi @monicah79, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
No linter errors✔️
The design looks good ✔️ 

## Required Changes ♻️
- [ ] Kindly use the Microverse readme template to document your work.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by monicah79** [comment]
hey sorry,,,thanks for the review i was really excited to pass and i
submitted immediately bt av worked on it

**Comment by monicah79** [comment]
passing was just a win for me i invested too much time i didnt think twice
when you pass it,,,

**Pull request merged**